lib/ui/onboarding/import_account/usecases/find_account_use_case.dart

```diff
@@ -30,7 +32,8 @@ class FindAccountsUseCase extends InputUseCase<Result<Iterable<UserProfileData>,
       final List<Result<ProfileData, HyphaError>> profiles = await Future.wait(futures);
 
       final List<UserProfileData> wtf = List.empty(growable: true);
-      for (final response in profiles) {
```

Comment:
handle case when there is no profile available

Reply:
This is good to know. I didnt know this was possible.

Reply:
Me neither ;)